e2e/rbd_helper.go

```diff
@@ -190,6 +190,46 @@ func getImageMeta(rbdImageSpec, metaKey string, f *framework.Framework) (string,
 	return strings.TrimSpace(stdOut), nil
 }
 
+func validateImageOwner(pvcPath string, f *framework.Framework) error {
+	pvc, err := loadPVC(pvcPath)
+	if err != nil {
+		return err
+	}
+	pvc.Namespace = f.UniqueName
+	pvc.Name = f.UniqueName
+	err = createPVCAndvalidatePV(f.ClientSet, pvc, deployTimeout)
+	if err != nil {
+		return err
+	}
+
+	imageData, err := getImageInfoFromPVC(pvc.Namespace, pvc.Name, f)
+	if err != nil {
+		return err
+	}
+
+	stdOut, stdErr, err := execCommandInToolBoxPod(f,
+		fmt.Sprintf("rados listomapkeys %s csi.volume.%s", rbdOptions(defaultRBDPool), imageData.imageID), rookNamespace)
+	if err != nil {
+		return err
+	}
+	if stdErr != "" {
+		return fmt.Errorf("failed to listomapkeys %v", stdErr)
+	}
+
+	if radosNamespace != "" {
+		e2elog.Logf("found image journal %s in pool %s namespace %s", "csi.volume."+imageData.imageID, defaultRBDPool, radosNamespace)
+	} else {
+		e2elog.Logf("found image journal %s in pool %s", "csi.volume."+imageData.imageID, defaultRBDPool)
+	}
+
+	ownerKey := "csi.volume.owner"
+	if !strings.Contains(stdOut, ownerKey) {
```

Comment:
can we validate the owner's name? instead of checking the key, best would be to validate the value we store

Reply:
Sure, will adapt it.